### PR TITLE
feat(cc-addon-admin): make use of the cc-notice instead of cc-error

### LINKED ISSUE #777: cc-addon-admin: make use of the cc-notice
Currently, we're moving our components that used  `cc-error` in their error state to `cc-notice`  to improve consistency.

However, some of them are tricky and we decided to handle `cc-addon-admin` in a separate issue/PR.

## Why?

In this component we have two error modes: 
* loading
* saving

The `loading` one is quite easy to switch, we just have to replace the component and that's pretty much it.
The `saving` one is however a bit more tricky.
As you can see from the screenshot below, the `cc-error` is on top of the rest of the component which is blurred on the background.

For various reasons it's not a good idea and we have decided to get rid of this system.
To resolve this issue, we have decided for this error mode to use a toast instead.

## How?

There are a few things we need to do to get this done:
* Make the `error` prop a boolean
* Switch the `cc-error` to a `cc-notice`
* Remove the loading case
* Add the toast (change to toast will need to be handled separately where we use this component internally)

### PR DESCRIPTION
**Pull request by Galimede**
Fixes #777

### CODE REVIEW

File: src/components/cc-addon-admin/cc-addon-admin.js [hunk 1]
```diff
@@ -97,7 +96,7 @@ export class CcAddonAdmin extends LitElement {
   render () {
 
     const isFormDisabled = (this.error !== false) || this.saving;
```
Comment (pdesoyres-cc) [inline src/components/cc-addon-admin/cc-addon-admin.js:98]: nitpick: minor simplification

```suggestion
    const isFormDisabled = this.error || this.saving;
```

**Review by pdesoyres-cc — CHANGES_REQUESTED** [review]
Well done! I just have one minor suggestion.

File: src/components/cc-addon-admin/cc-addon-admin.js [hunk 2]
```diff
@@ -147,17 +146,7 @@ export class CcAddonAdmin extends LitElement {
...
-
-        ${this.saving ? html`
-          <cc-loader slot="overlay"></cc-loader>
-        ` : ''}
-
-        ${this.error === 'saving' ? html`
-          <div slot="overlay">
-            <cc-error mode="confirm" @cc-error:ok=${this._onDismissError}>${i18n('cc-addon-admin.error-saving')}</cc-error>
-          </div>
+          <cc-notice intent="warning" .message="${i18n('cc-addon-admin.error-loading')}"></cc-notice>
```
Comment (florian-sanders-cc) [inline src/components/cc-addon-admin/cc-addon-admin.js:149]: **nitpick:** 
```suggestion
          <cc-notice intent="warning" message="${i18n('cc-addon-admin.error-loading')}"></cc-notice>
```

**Review by florian-sanders-cc — APPROVED** [review]
Gg @Galimede apart from a small nitpick (the usual one :joy:) it's all good!

**Review by hsablonniere — APPROVED** [review]
:+1: All good for me, well done!

**Review by roberttran-cc — APPROVED** [review]
LGTM: perfect!

**Pull request merged**